tests/core/vm/test_vm.py

```diff
@@ -59,8 +59,12 @@ def test_state_db(chain_without_block_validation):  # noqa: F811
     with vm.state_db() as state_db:
         pass
 
-    with pytest.raises(TypeError):
-        state_db.increment_nonce(address)
+    # XXX: This test used to pass because the original _get_account() code calls
+    # self._trie[address], and that raises a TypeError since _trie is None, but that makes the
```

Comment:
A specific exception would be my preference, but this is no longer relevant as I'm dropping that commit as you've suggested. Will leave it in another branch and link to it from the issue, in case we want to reconsider it in the future